frontend/app/services/TouchpointBackend.scala

```diff
@@ -85,21 +85,7 @@ object TouchpointBackend {
     val paymentService = new PaymentService(zuoraSoapService, newCatalogService.unsafeCatalog.productMap)
     val salesforceService = new SalesforceService(config.salesforce)
     val identityService = IdentityService(new IdentityApi(wsClient, executionContext))
-    val memberService = new MemberService(
-      identityService = identityService,
-      salesforceService = salesforceService,
-      zuoraService = zuoraSoapService,
-      zuoraRestService = zuoraRestService,
-      ukStripeService = stripeUKMembershipService,
-      auStripeService = stripeAUMembershipService,
-      subscriptionService = newSubsService,
-      catalogService = newCatalogService,
-      paymentService = paymentService,
-      discounter = discounter,
-      discountIds = Config.discountRatePlanIds(config.zuoraEnvName),
-      invoiceIdsByCountry = Config.invoiceTemplateOverrides(config.zuoraEnvName),
-      ec = executionContext
-    )
+    val memberService = new MemberService()(executionContext)
```

Comment:
member service (above file) has been simplified to the point of not really having much code of use, therefore almost no dependencies.